SpinCore_pp/pulse_length_conv.py

```diff
@@ -3,75 +3,153 @@
 from pyspecdata import r_, nddata
 
 
-def prog_plen(desired_actual):
+def prog_plen(desired_actual, amplitude=1.0):
     """
-    Takes the desired pulse length and tells the
+    Takes the desired beta (us sqrt(W)) and tells the
     user what pulse length should be programmed in order to get the actual desired
-    pulse length
+    beta
     Parameters
     ==========
     desired_actual: float
-                    the actual pulse length you wish the spincore to output,
-                    in us
+                    the actual beta you wish the spincore to output,
```

Comment:
Can we call this parameter beta, desired beta, or just desired. Desired actual is very weird... I don't know what it means